src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -88,70 +72,105 @@ public static void main(String[] args) {
         }
     }
 
-    /**
-     * Set up the Vert.x metrics options
-     *
-     * @return instance of the MicrometerMetricsOptions on Vert.x
-     */
-    private static MicrometerMetricsOptions metricsOptions() {
-        Set<String> set = new HashSet<>();
-        set.add(MetricsDomain.NAMED_POOLS.name());
-        set.add(MetricsDomain.VERTICLES.name());
-        return new MicrometerMetricsOptions()
-                .setPrometheusOptions(new VertxPrometheusOptions().setEnabled(true))
-                // define the labels on the HTTP server related metrics
-                .setLabels(EnumSet.of(Label.HTTP_PATH, Label.HTTP_METHOD, Label.HTTP_CODE))
-                // disable metrics about pool and verticles
-                .setDisabledMetricsCategories(set)
-                .setJvmMetricsEnabled(true)
-                .setEnabled(true);
-    }
-
     /**
      * Deploys the HTTP bridge into a new verticle
      *
-     * @param vertx                 Vertx instance
      * @param bridgeConfig          Bridge configuration
-     * @param metricsReporter       MetricsReporter instance for scraping metrics from different registries
      * @return                      Future for the bridge startup
      */
-    private static Future<HttpBridge> deployHttpBridge(Vertx vertx, BridgeConfig bridgeConfig, MetricsReporter metricsReporter)  {
+    private static Future<HttpBridge> deployHttpBridge(BridgeConfig bridgeConfig) 
+            throws MalformedObjectNameException, IOException {
         Promise<HttpBridge> httpPromise = Promise.promise();
 
-        HttpBridge httpBridge = new HttpBridge(bridgeConfig, metricsReporter);
+        Vertx vertx = createVertxInstance(bridgeConfig);
+        MetricsCollector metricsCollector = getMetricsCollector(bridgeConfig);
```

Comment:
What is the reason for creating the `metricsCollector` here and not inside `HttpBridge`? It seems to have the BridgeConfig instance and all what it is used for here seems to be the log message which can be surely embedded there as well. (I guess there might be something, but it does not seem obvious)